RESTore/ExecutionContext.cs

```diff
@@ -52,6 +52,11 @@ public class ExecutionContext
         /// </summary>
         private HttpClient _httpClient;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        private HttpWebRequest _webRequest;
+
```

Comment:
empty summary here Kev

Reply:
Added as part of the performance pull request :)